src/components/pages/Offers/Offer/OfferDetails/OfferForm/OfferCreation.jsx

```diff
@@ -46,7 +46,7 @@ const OfferCreation = ({
         const offerersRequest = pcapi.getUserValidatedOfferersNames().then(receivedOfferers => {
           offerersNames.current = receivedOfferers
         })
-        const venuesRequest = pcapi.getVenuesForOfferer().then(receivedVenues => {
+        const venuesRequest = pcapi.getVenuesForOfferer(null, true).then(receivedVenues => {
```

Comment:
J'aime pas trop avoir des paramètres `null` et `true`, à la lecture je ne vois pas à quoi ça correspond. Est-ce que ça ne vaudrait pas le coup d'avoir un seul paramètre `options` qui serait un objet avec différentes clés optionnelles (offererId, activeOfferersOnly...)
On appellerait donc ici : 
```suggestion
        const venuesRequest = pcapi.getVenuesForOfferer({ activeOfferersOnly: true }).then(receivedVenues => {
```

Reply:
On peut rajouter un test aussi côté composant pour s'assurer que quand pas de structure sélectionnée, on appelle `getVenuesForOfferer` avec le bon paramètre